src/components/NetworkSelector/index.tsx

```diff
@@ -194,6 +197,10 @@ export const NetworkSelector = () => {
       const latestData = getData();
 
       selectNetwork(latestData);
+
+      // also update the network setting for endpoints
+      updateNetwork(latestData);
```

Comment:
added this to update its global endpoints network setting so that users would see the change right away vs. refresh the page to see the updated endpoint